### ISSUE TITLE
Error: JSON errors when running Kover with the IntelliJ coverage engine

### ISSUE THREAD

**Issue by cristan** [Kotlin/kotlinx-kover#77]
**Steps to reproduce** (or use [the attached project](https://github.com/Kotlin/kotlinx-kover/files/7609152/TestKoverAndroidProject.zip))
* Create an empty Android Project in Android studio.
* Add the following after the buildscript block in the root build.gradle
```
plugins {
    id 'org.jetbrains.kotlinx.kover' version '0.4.2'
}
```

* Add the following in app/build.gradle:

```
kover {
    coverageEngine.set(kotlinx.kover.api.CoverageEngine.INTELLIJ)
    intellijEngineVersion.set('1.0.639') // The version has to be specified, because the default version (622) isn't on Maven Central https://github.com/Kotlin/kotlinx-kover/issues/76
}
```
* Change the repositoriesMode to RepositoriesMode.PREFER_SETTINGS (because [this change](https://github.com/Kotlin/kotlinx-kover/issues/34#issuecomment-963161529) isn't in 0.4.2 yet)

* Add a class and create a test
* Clean the project
* Run the gradle tasks `koverVerify koverHtmlReport`

You then see this error:
```
com.intellij.rt.coverage.report.ReporterArgs$ArgParseException: Incorrect arguments in file /Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/tmp/koverHtmlReport/intellijreport.args
	at com.intellij.rt.coverage.report.ReporterArgs.from(ReporterArgs.java:66)
	at com.intellij.rt.coverage.report.Main.main(Main.java:29)
Caused by: org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:507)
	at org.json.JSONObject.<init>(JSONObject.java:222)
	at org.json.JSONObject.<init>(JSONObject.java:406)
	at com.intellij.rt.coverage.report.ReporterArgs.parse(ReporterArgs.java:72)
	at com.intellij.rt.coverage.report.ReporterArgs.from(ReporterArgs.java:62)
	... 1 more
/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/tmp/koverHtmlReport/intellijreport.args

Arguments must be passed in the following JSON format:
{
  "modules": [
    { "reports": [
        {"ic": "path to ic binary file", "smap": "path to source map file"}
      ] [OPTIONAL, absence means that all classes were not covered],
      "output": ["outputRoot1", "outputRoot2"],
      "sources": ["sourceRoot1", "sourceRoot2"]
    }
  ],
  "xml": "path to xml file" [OPTIONAL],
  "html": "path to html directory" [OPTIONAL]
Caused by: org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]

}

Execution failed for task ':app:koverHtmlReport'.
> Process 'command '/Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java'' finished with non-zero exit value 1
```

The value of intellijreports.args:
```
/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/kover/testDebugUnitTest.ic
/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/kover/testDebugUnitTest.ic.smap
/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/kover/testReleaseUnitTest.ic
/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/kover/testReleaseUnitTest.ic.smap

/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/src/main/java

/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/tmp/kotlin-classes/release
/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/tmp/kotlin-classes/debug
/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/intermediates/javac/release/classes
/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/intermediates/javac/debug/classes


/Users/cristan/AndroidStudioProjects/TestKoverAndroidProject/app/build/reports/kover/html
```
As you can see, this is no JSON, so it indeed can't be parsed as such.

**Comment by shanshin** [comment]
Hi,
the version of the IntelliJ agent published in Maven Central is incompatible with the version of Kover 0.4.2.
In the next release, it will be possible to use the IntelliJ agent, which are now published in Maven Central.

**Comment by shanshin** [comment]
Duplicate #34

**Comment by shanshin** [comment]
Fixed in `0.4.3`
